app.ts

```diff
@@ -0,0 +1,462 @@
+const headings: any = document.querySelector("#Headings");
+const content_cols: any = document.querySelector("#Content");
+const pageStats: any = document.getElementById("pageStats");
+const nextButton: any = document.querySelector("#next");
+const prevButton: any = document.querySelector("#prev");
+const clear = "";
+let paramOne: any = "0";
+let paramTwo: any = "9";
+let contentNeeded: any = [];
+
+// Fetch requests
+
+//// Heading Row(The Columns)
+
+function getColumns() {
+  fetch("http://localhost:2050/columns", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((res) => res.text())
+    .then((data) => {
+      data = JSON.parse(data);
+      let colList = data;
+      for (let i = 0; i < colList.length; i++) {
+        colHeading(colList[i]);
+      }
+    });
+}
+
+//// Table Content
+
+function getTable() {
+  fetch("http://localhost:2050/records?from=" + paramOne + "&to=" + paramTwo, {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((res) => res.text())
+    .then((data) => {
+      data = JSON.parse(data);
+      let contentList = data;
+      for (let i = 0; i < contentList.length; i++) {
+        let contentListContent: any = contentList[i];
+        cols(contentListContent);
+      }
+    });
+}
+
+//// Display the current shown results
+
+function stats() {
+  fetch("http://localhost:2050/recordCount", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((res) => res.text())
+    .then((data) => {
+      data = JSON.parse(data);
+      let count = data;
+      let currentStats =
+        "Showing results from " +
+        paramOne +
+        " to " +
+        paramTwo +
+        " out of " +
+        count +
+        " results.";
+      pageStats.innerHTML = currentStats;
+    });
+}
+
+// Loads all intial fields/data
+
+window.onload = function () {
+  let windowHeight = window.innerHeight;
+
+  if (windowHeight < 600 && windowHeight >= 480) {
```

Comment:
Hmmm, I would avoid doing this check: again it ties into the calculation to fit as much readable rows on the screen. So rather have a function that calculates and returns the number of rows you can fit on the page and then use that value to set your parameters.